kafka-ui-react-app/src/components/Topics/List/ListHeader.tsx

```diff
@@ -0,0 +1,73 @@
+import { TopicColumnsToSort } from 'generated-sources';
+import React from 'react';
+import cx from 'classnames';
+
+export interface ListHeaderProps {
+  orderBy: TopicColumnsToSort | undefined;
+  setOrderBy: React.Dispatch<
+    React.SetStateAction<TopicColumnsToSort | undefined>
+  >;
+}
+
+const ListHeader: React.FC<ListHeaderProps> = ({ orderBy, setOrderBy }) => {
+  return (
+    <thead>
+      <tr>
+        <th
+          className={cx(
+            'is-clickable',
+            orderBy === TopicColumnsToSort.NAME && 'has-text-link-dark'
+          )}
+          onClick={() => setOrderBy(TopicColumnsToSort.NAME)}
+        >
+          Topic Name{' '}
+          <span className="icon is-small">
+            {orderBy === TopicColumnsToSort.NAME ? (
+              <i className="fas fa-caret-up" />
+            ) : (
+              <i className="fas fa-caret-down" />
+            )}
```

Comment:
Usually `caret-up` indicates that data is sorted in ascending order, and `caret-down` - in descending. You can use `fa-sort` icon to indicate that column is sortable.